### PR TITLE
change path of define.ts in gfx-define-generator

### PR DESCRIPTION
**Pull request by yiwenxue**
Re: #

### Changelog

* change path of define.ts in gfx-define-generator
* sync the definition of SwapchainInfo

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: cocos/gfx/base/define.ts [hunk 1]
```diff
@@ -1012,42 +1012,19 @@ export class BindingMappingInfo {
...
-        this.windowHandle = info.windowHandle;
-        return this;
-    }
-}
-
 export class SwapchainInfo {
     declare private _token: never; // to make sure all usages must be an instance of this exact class, not assembled from plain object
 
     constructor (
+        public windowId: number = 0,
```
Comment (star-e) [inline cocos/gfx/base/define.ts:1019]: This file is generated from c++ using gfx-define-generator. TS windowId is now synchronized with c++ implementation.

File: cocos/gfx/base/define.ts [hunk 2]
```diff
@@ -1012,42 +1012,19 @@ export class BindingMappingInfo {
...
-        this.x = info.x;
-        this.y = info.y;
-        this.width = info.width;
-        this.height = info.height;
-        this.flags = info.flags;
-        this.windowHandle = info.windowHandle;
-        return this;
-    }
-}
-
```
Comment (star-e) [inline cocos/gfx/base/define.ts:1039]: In c++ code, `SystemWindowInfo` is not in gfx defines. So when generating ts code, it is removed.

File: cocos/gfx/device-manager.ts [hunk 3]
```diff
@@ -161,7 +161,7 @@ export class DeviceManager {
     }
 
     private _initSwapchain () {
-        const swapchainInfo = new SwapchainInfo(this._canvas!);
+        const swapchainInfo = new SwapchainInfo(0, this._canvas!);
```
Comment (star-e) [inline cocos/gfx/device-manager.ts]: Web has only one window. Is 0 a valid windowId？ @minggo

**Review by star-e — APPROVED** [review]

File: cocos/gfx/device-manager.ts (hunk 3 above)
Comment (minggo) [reply cocos/gfx/device-manager.ts]: Valid windowId start from 1.

Comment (minggo) [reply cocos/gfx/device-manager.ts]: Valid windowId start from 1.

**Review by minggo — APPROVED** [review]

**Pull request merged**